src/WebSocketServer.php

```diff
@@ -74,6 +78,18 @@ public function __construct(
         $this->setIsPcntlLoaded(extension_loaded('pcntl'));
     }
 
+    /**
+     * Set the looping sleep in microseconds
+     *
+     * @return self
+     */
+    public function loopingDelay(int $loopingDelay): self
```

Comment:
It would be better to set up some thresholds min/max, because if user will set up say 1 it will be too low to make things better for CPU and rather consume more for nothing and vice versa 10 000 000 let it sleep for 10 seconds in the loop just for nothing. Also, is it a good idea to let users choose this amount of delay? Probably constant value will be more of a fit here?

Reply:
Yes, we can add some fixed thresholds :)

Reply:
And we can ask for milliseconds instead of millionths of a second.